NOTES.create-cmd_states.db3

```diff
@@ -1,4 +1,9 @@
-To make the fresh sqlite3 version of relevant command states Sybase tables:
+To make a fresh sqlite3 version of command states Sybase tables:
 
-$ ./make_new_tables.py --tstart=1999:001 --cmds --cmd-states  # takes a few minutes
+$ ./make_new_tables.py --tstart=1999:001 --cmds --cmd-states # takes a few minutes
+
+To make a complete sqlite3 version to use for on-the-side sqlite3 crontasks, do:
+
+# ./make_new_tables.py --tstart=1999:001 --cmds --cmd-states --tl_processing # takes a few minutes
```

Comment:
> If you have no reason to want to just copy cmd_states and timelines, I'll cut that from the notes.

Indeed I have no reason to want just the cmd_states and cmds and timelines.  That was a result of my ignorance of those `tl_*` tables, which to this day I do not really understand.